__tests__/articles.test.js

```diff
@@ -0,0 +1,50 @@
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data/index");
+
+beforeEach(() => seed(testData));
+afterAll(() => db.end());
+
+describe("/api/articles/:article_id", () => {
+  test("200: responds with an article object with the correct properties", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article).toBeInstanceOf(Object);
+
+        expect(body.article).toEqual(
+          expect.objectContaining({
+            author: expect.any(String),
+            title: expect.any(String),
+            article_id: expect.any(Number),
+            body: expect.any(String),
+            topic: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            article_img_url: expect.any(String),
+          })
```

Comment:
We should test for specifics here. E.g. article_id should be 1 not any Number because that's what we're asking for here 😄

Reply:
thanks I have changed this to 1 :)